spatial-data-viz.Rmd

```diff
@@ -1,1328 +1,1372 @@
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/spatial_data_viz/gps15.png')
+```
+
+16. Set the *Frame rate* to `10`.
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/spatial_data_viz/gps16.png')
+```
+
+17. Check the *Loop* button and hit the *Play* button. You will see the map canvas animate to show the trip progress.
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/spatial_data_viz/gps17.png')
+```
+
+18. It would be useful to have the current time displayed on the map. Go to **View &rarr; Decorations &rarr; Title Label...**.
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/spatial_data_viz/gps18.png')
+```
+
+19. In the *Title Label Decoration*, click the *Insert an Expression* button. 
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/spatial_data_viz/gps19.png')
+```
+
+20. The current timestamp of the map is stored in the `@map_start_time` variable. We can use it with the `format_date()` function to create a readable timestamp. But note that the GPS timestamps are in universal time (UTC). So we can use `to_interval()` function to convert it to the UTC+5:30 timezone for India. Enter the following expression
+
+```
+format_date( @map_start_time + to_interval('5 hours 30 mins'), 'yyyy-MM-dd hh:mm')
+```
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/spatial_data_viz/gps20.png')
+```
+
+21. Click the *Font* options and increase the font size to `24`. Set the *Background bar color* to `White`. Click *OK*.
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/spatial_data_viz/gps21.png')
+```
+
+22. Now as you play the animation, the timestamp will update to display the time of the current point on the track. Click the *Export Animation* button to save the animation as individual frames.
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/spatial_data_viz/gps22.png')
+```
+
+23. In the *Export Map Animation* dialog, select the *Output directory*. The track is over 500 seconds long, so to reduce the number of frames, you can increate the *Step* to `5`. Click *Save* and QGIS will write an image for each time step to the chosen directory.
+
+```{r echo=FALSE, fig.align='center', out.width='60%'}
+knitr::include_graphics('images/spatial_data_viz/gps23.png')
+```
+
+24. Once you have the individual frames, you can use a program such as [ezgif.com](https://ezgif.com/) to create an animated GIF from them.
+
+```{r echo=FALSE, fig.align='center', out.width='75%'}
+knitr::include_graphics('images/spatial_data_viz/gps24.png')
+```
+

```

Comment:
Add the resulting GIF (file is in the aciwrm folder) and link it as it is done previously

Reply:
GIF linked.